comps/comps-foreman-plugins-el8.xml

```diff
@@ -231,12 +234,14 @@
       <packagereq type="default">rubygem-dry-validation-doc</packagereq>
       <packagereq type="default">rubygem-ed25519-doc</packagereq>
       <packagereq type="default">rubygem-elastic-apm-doc</packagereq>
+      <packagereq type="default">rubygem-et-orbi-doc</packagereq>
       <packagereq type="default">rubygem-faraday-cookie_jar-doc</packagereq>
       <packagereq type="default">rubygem-faraday-detailed_logger-doc</packagereq>
       <packagereq type="default">rubygem-faraday_middleware-doc</packagereq>
       <packagereq type="default">rubygem-ffi-compiler-doc</packagereq>
       <packagereq type="default">rubygem-fog-kubevirt-doc</packagereq>
       <packagereq type="default">rubygem-fog-proxmox-doc</packagereq>
+      <packagereq type="default">rubygem-foreman-tasks-doc</packagereq>
```

Comment:
huh, missed that one in the previous review.
there is already an entry for `rubygem-foreman-tasks-doc` in L279, so that one seems superfluous?

Reply:
Huh, not sure how it got there. Removed